docs/containers.conf.5.md

```diff
@@ -310,6 +310,15 @@ pod consumes one lock.  The default number available is 2048.  If this is
 changed, a lock renumbering must be performed, using the
 `podman system renumber` command.
 
+**remote_uri**=""
+  URI to access Podman service
+
+- **rootless local**  - unix://run/user/$UID/podman/podman.sock
+- **rootless remote** - ssh://engineering.lab.company.com/run/user/1000/podman/podman.sock
+- **rootfull local**  - unix://run/podman/podman.sock
+- **rootfull remote** - ssh://root@10.10.1.136:22/run/podman/podman.sock
```

Comment:
is this IP correct for all?

Reply:
Nope just trying to show examples or how you could configure this.

Reply:
Wish we could generalize it, but tthe couple of ways I tried were gross looking and this is under an "Examples URIs" title, so let's leave it.  Thx for the 411 @rhatdan